### ISSUE TITLE
Sliders value is calculated badly

### ISSUE THREAD

**Issue by Fifaldi** [Sharcoux/slider#91]
**Describe the bug**
when minimumValue is not divisible by step then values are calculated incorrectly, as if they were calculated from 0 not from minimum value.
 
**To Reproduce**
-https://snack.expo.dev/@fifaldi/sharcoux-slider-demo
Or create following comopnent:
```
import * as React from 'react';
import { Text, View } from 'react-native';
import { Slider } from '@sharcoux/slider'

export default function App() {
  const [state, setState] = React.useState(1)
  return (
    <View >
      <Text>Minimum is 1, Maximum is 13, step is 2 values should be: 1/3/5/7/9/11/13</Text>
      <Text>Values received: 1/2/4/6/8/10/12/13</Text>
      <Slider
        value={state} 
        minimumValue={1}
        maximumValue={13}
        step={2}
        onValueChange={setState}
      />
      <Text>Normal slider value: {state}</Text>
    </View>
  );
}
```
Example sliders with bug:
 <Slider minimumValue={1} maximumValue={13} step={2} />
 <Slider minimumValue={10} maximumValue={19} step={3} />
 <Slider minimumValue={3} maximumValue={12} step={4} />

**Expected behavior**
Values are calculated staring from minimumValue by adding step

**Screenshots**
<img width="339" alt="Screenshot 2024-01-29 at 16 34 09" src="https://github.com/Sharcoux/slider/assets/45095503/776c3f7a-866a-498c-bce1-1f2c4c37461f">
<img width="344" alt="Screenshot 2024-01-29 at 16 34 14" src="https://github.com/Sharcoux/slider/assets/45095503/136a42ba-d7aa-43f8-a18e-322524c8f30e">
<img width="340" alt="Screenshot 2024-01-29 at 16 34 18" src="https://github.com/Sharcoux/slider/assets/45095503/a799aae7-79f9-4ae1-9cac-8ede4d81fd2c">

**Please complete the following information:**
 - OS: iOS, Android, Browser
 - Browser: chrome
 - @react-native-assets/slider version:  7.0.7

**Comment by Sharcoux** [comment]
Confirmed. On it.

**Comment by Sharcoux** [comment]
Solved in version 7.1.0

Don't hesitate to come back if any problem persist.
